From 436e7a406724efa98d05d4c32cae027f31a66033 Mon Sep 17 00:00:00 2001
From: Laszlo Ersek <lersek@redhat.com>
Date: Fri, 23 Sep 2016 14:39:35 -0300
Subject: [PATCH 1/2] hw/i386: regenerate checked-in AML payload (RHEL only)

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <20160923143936.25594-2-lersek@redhat.com>
Patchwork-id: 72414
O-Subject: [RHEL-7.3 qemu-kvm PATCH 1/2] hw/i386: regenerate checked-in AML payload (RHEL only)
Bugzilla: 1377087
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>

In preparation for the next patch, which will flip the build from IASL to
pre-generated AML, rebuild the checked-in AML content, with the last iasl
version that's known -- from Brew -- to generate good AML, namely
"acpica-tools-20150619-3.el7.x86_64".

Only checksums and likely some metadata change in the AML payload. This
proves that our checked-in files have been up to date, and it's safe to
switch the build to them. I actually verified this in a RHEL-7 guest, with
"acpidump -b" and "iasl -d" -- the dumped and disassembled DSL files
remained identical across this change.

Signed-off-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/i386/acpi-dsdt.hex.generated     |  8 ++++----
 hw/i386/q35-acpi-dsdt.hex.generated |  8 ++++----
 hw/i386/ssdt-misc.hex.generated     |  8 ++++----
 hw/i386/ssdt-pcihp.hex.generated    |  8 ++++----
 hw/i386/ssdt-proc.hex.generated     | 12 ++++++------
 5 files changed, 22 insertions(+), 22 deletions(-)

diff --git a/hw/i386/acpi-dsdt.hex.generated b/hw/i386/acpi-dsdt.hex.generated
index 2c01107..b2af8d0 100644
--- a/hw/i386/acpi-dsdt.hex.generated
+++ b/hw/i386/acpi-dsdt.hex.generated
@@ -8,7 +8,7 @@ static unsigned char AcpiDsdtAmlCode[] = {
 0x0,
 0x0,
 0x1,
-0xe0,
+0xea,
 0x42,
 0x58,
 0x50,
@@ -31,9 +31,9 @@ static unsigned char AcpiDsdtAmlCode[] = {
 0x4e,
 0x54,
 0x4c,
-0x23,
-0x8,
-0x13,
+0x19,
+0x6,
+0x15,
 0x20,
 0x10,
 0x49,
diff --git a/hw/i386/q35-acpi-dsdt.hex.generated b/hw/i386/q35-acpi-dsdt.hex.generated
index 32c16ff..23ab305 100644
--- a/hw/i386/q35-acpi-dsdt.hex.generated
+++ b/hw/i386/q35-acpi-dsdt.hex.generated
@@ -8,7 +8,7 @@ static unsigned char Q35AcpiDsdtAmlCode[] = {
 0x0,
 0x0,
 0x1,
-0x6,
+0x10,
 0x42,
 0x58,
 0x50,
@@ -31,9 +31,9 @@ static unsigned char Q35AcpiDsdtAmlCode[] = {
 0x4e,
 0x54,
 0x4c,
-0x23,
-0x8,
-0x13,
+0x19,
+0x6,
+0x15,
 0x20,
 0x10,
 0x49,
diff --git a/hw/i386/ssdt-misc.hex.generated b/hw/i386/ssdt-misc.hex.generated
index 55e3bd2..86c5725 100644
--- a/hw/i386/ssdt-misc.hex.generated
+++ b/hw/i386/ssdt-misc.hex.generated
@@ -23,7 +23,7 @@ static unsigned char ssdp_misc_aml[] = {
 0x0,
 0x0,
 0x1,
-0x76,
+0x80,
 0x42,
 0x58,
 0x50,
@@ -46,9 +46,9 @@ static unsigned char ssdp_misc_aml[] = {
 0x4e,
 0x54,
 0x4c,
-0x23,
-0x8,
-0x13,
+0x19,
+0x6,
+0x15,
 0x20,
 0x10,
 0x42,
diff --git a/hw/i386/ssdt-pcihp.hex.generated b/hw/i386/ssdt-pcihp.hex.generated
index b3c2cd5..2f946d5 100644
--- a/hw/i386/ssdt-pcihp.hex.generated
+++ b/hw/i386/ssdt-pcihp.hex.generated
@@ -17,7 +17,7 @@ static unsigned char ssdp_pcihp_aml[] = {
 0x0,
 0x0,
 0x1,
-0x76,
+0x80,
 0x42,
 0x58,
 0x50,
@@ -40,9 +40,9 @@ static unsigned char ssdp_pcihp_aml[] = {
 0x4e,
 0x54,
 0x4c,
-0x23,
-0x8,
-0x13,
+0x19,
+0x6,
+0x15,
 0x20,
 0x10,
 0x33,
diff --git a/hw/i386/ssdt-proc.hex.generated b/hw/i386/ssdt-proc.hex.generated
index bb9920d..1b4118a 100644
--- a/hw/i386/ssdt-proc.hex.generated
+++ b/hw/i386/ssdt-proc.hex.generated
@@ -11,7 +11,7 @@ static unsigned char ssdp_proc_aml[] = {
 0x0,
 0x0,
 0x1,
-0xb8,
+0x82,
 0x42,
 0x58,
 0x50,
@@ -34,9 +34,9 @@ static unsigned char ssdp_proc_aml[] = {
 0x4e,
 0x54,
 0x4c,
-0x23,
-0x8,
-0x13,
+0x19,
+0x6,
+0x15,
 0x20,
 0x5b,
 0x83,
@@ -47,8 +47,8 @@ static unsigned char ssdp_proc_aml[] = {
 0x41,
 0x41,
 0xaa,
-0x10,
-0xb0,
+0x0,
+0x0,
 0x0,
 0x0,
 0x0,
-- 
1.8.3.1

